systemvm/patches/debian/config/opt/cloud/bin/cs/CsNetfilter.py

```diff
@@ -150,6 +150,8 @@ def compare(self, list):
             new_rule.set_table(fw[0])
             if isinstance(fw[1], int):
                 new_rule.set_count(fw[1])
+
+            logging.debug("Checking if the rule already exists: rule=%s table=%s chain=%s", new_rule.get_rule(), new_rule.get_table(), new_rule.get_chain())
```

Comment:
We really need some logging, during dev or prod, to be able to see what is going on. Once we get more confidence with the current implementation, we can remove some logging.